doc/cookbook/api-aggregation.md

```diff
@@ -183,8 +183,7 @@ flow:
   namespace: demo3
 - filter: proxy-demo1
   namespace: demo1
-  jumpIf:
-    serverError: "proxy-demo2"
+  jumpIf: { serverError: proxy-demo2 }
```

Comment:
why change this line? I think it is correct both before and after the change.

Reply:
> why change this line? I think it is correct both before and after the change.

That's right, both are correct. Just want to keep the same code style with other cookbook examples
e.g. in [Pipeline](https://github.com/megaease/easegress/blob/main/doc/cookbook/pipeline.md#built-in-filter-end): 
```yaml
- filter: validator
  jumpIf: { invalid: buildFailureResponse }
```
If there's no necessary, I can recover it.

Reply:
ok, that's fine.